pom.xml

```diff
@@ -181,8 +182,8 @@
     <kuali-mvn.version>1.0.2</kuali-mvn.version>
     <log4j.version>1.2.16</log4j.version>
     <mail.version>1.4.4</mail.version>
-    <mockito.version>1.10.8</mockito.version>
-    <powermock.version>1.6.1</powermock.version>
+    <mockito.version>2.23.0</mockito.version>
+    <powermock.version>2.0.0-RC.4</powermock.version>
```

Comment:
The full release of this version of Powermock is still not available so bringing in the most recent release candidate.